pkg/controller/populators/import-populator.go

```diff
@@ -156,17 +156,21 @@ func (r *ImportPopulatorReconciler) reconcileTargetPVC(pvc, pvcPrime *corev1.Per
 			break
 		}
 
-		// Once the import is succeeded, we rebind the PV from PVC' to target PVC
-		if err := cc.Rebind(context.TODO(), r.client, pvcPrime, pvc); err != nil {
-			return reconcile.Result{}, err
+		if cc.IsPVCComplete(pvcPrime) {
```

Comment:
I'd add here `&& cc.IsUnbound(pvc)` to avoid running Rebind when the PVC is bound.

Reply:
ACK